consensus/actor_state.go

```diff
@@ -0,0 +1,97 @@
+package consensus
```

Comment:
This seems pretty independent of consensus. I think we need a package for the combination of chain and state primitives into things like this. I don't think the right package exists yet, so I'm flagging here that we want to make room for that. cc @icorderi